src/presenters/admin/sidebar-presenter.php

```diff
@@ -89,7 +89,11 @@ class="attachment-full size-full content-visible"
 						</p>
 						<p class="yoast-price-micro-copy">
 							<?php
-								echo \esc_html__( 'Only $/€/£99 per year (ex VAT) 30-day money back guarantee - No questions asked.', 'wordpress-seo' );
+								echo \esc_html__( 'Only $/€/£99 per year (ex VAT)', 'wordpress-seo' );
+							?>
+							<br />
+							<?php
+								echo \esc_html__( '30-day money back guarantee.', 'wordpress-seo' );
```

Comment:
That might not be how languages work? In the JS one translators get to translate the full suggestion. It seems to me that is preferable if it is actually one sentence.